kubernetes-ingress/values.yaml

```diff
@@ -266,6 +266,10 @@ controller:
     pollingInterval: 30
     cooldownPeriod: 300
     restoreToOriginalReplicaCount: false
+    fallback: {}
```

Comment:
Block does not appear conforming to the specification. Specification states that failureThreshold and replicas fields are mandatory if fallback block is enabled, so it is either whole block is enabled or whole block is disabled (ie. commented out).